packages/trader/src/Modules/SmartChart/Components/all-markers.jsx

```diff
@@ -214,6 +255,7 @@ const shadowed_text = ({ ctx, color, is_dark_theme, text, left, top, scale }) =>
     for (let i = 0; i < (is_firefox ? 1 : 5); ++i) {
         ctx.fillText(text, left, top);
     }
+    ctx.restore();
```

Comment:
this ctx.save and ctx.restore fixes a production issue when everything on the chart gets blurry when switching between themes with this shadowed_text being present on the chart.